### PR TITLE
Add digital clock exercise

### PR DESCRIPTION
**Pull request by kariabancroft**
@jnf let me know if you think this is enough info for completion

### CODE REVIEW

File: topic_resources/javascript/digital-clock.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Digital Clock
+
+### Create a digital clock in your browser!
+
+### Getting Started
+- Create a function which will do all of the work to calculate and show the time. Start by loading this function using the document ready.
+
+  ```
+  $(document).ready( function() {
```
Comment (jnf) [inline topic_resources/javascript/digital-clock.md]: We'd need jQuery to do this, yeah? I think we'd need to use something like
```javascript
document.addEventListener('DOMContentLoaded', function(){
 // ...
});
```

Comment (kariabancroft) [inline topic_resources/javascript/digital-clock.md]: Yeah I wasn't sure if we wanted to just throw that one piece in there first, or use this. Either way it's probably a knowledge leap

Comment (jnf) [inline topic_resources/javascript/digital-clock.md]: What if we just provide them the static html file and/or talk about loading js at the end of the file? As long as the `<script`> tag is at the end of the file (and there's no dynamic content), then the dom would be ready when the script executed.

**Comment by kariabancroft** [comment]
Not merging - using a diff repo

**Pull request closed without merging**